packages/bot/src/components/dialog.jsx

```diff
@@ -58,9 +58,12 @@ const Dialog = ({
 Dialog.propTypes = {
     children           : PropTypes.node,
     is_open            : PropTypes.bool,
-    onCancelButtonClick: PropTypes.func,
-    onOkButtonClick    : PropTypes.func,
-    title              : PropTypes.string,
+    onCancelButtonClick: PropTypes.oneOfType([
+        PropTypes.bool,
```

Comment:
@aaron-binary why we need bool? if onCancelButtonClick is undefined it wont show the button